src/DependencyInjection/CmfRoutingExtension.php

```diff
@@ -50,6 +50,7 @@ private function setupChainRouter(array $config, ContainerBuilder $container, Lo
         $loader->load('routing-chain.xml');
 
         $container->setParameter('cmf_routing.replace_symfony_router', $config['chain']['replace_symfony_router']);
+        $container->setParameter('cmf_routing.redirectable_url_matcher', $config['chain']['redirectable_url_matcher']);
```

Comment:
when you move the flag to the dynamic router configuration, the key in $config will change.